### PR TITLE
[Zilliqa] small batch json rpc requests

### LINKED ISSUE #1406: [Zilliqa] 413 error when requesting large blocks
## Summary of Bug

```
ERRO[0171] Client Errors                                 fingerprint="[client_errors]" tags="[{status_code 413} {host api.zilliqa.com} {path }]" url="https://api.zilliqa.com"
WARN[0171] retry GetBlockByNumber                        attempts=4 number=1002025 sleep=5.205359519s symbol=ZIL
ERRO[0176] Client Errors                                 fingerprint="[client_errors]" tags="[{status_code 413} {host api.zilliqa.com} {path }]" url="https://api.zilliqa.com"
WARN[0177] retry GetBlockByNumber                        attempts=3 number=1002025 sleep=11.17723038s symbol=ZIL
ERRO[0188] Client Errors                                 fingerprint="[client_errors]" tags="[{status_code 413} {host api.zilliqa.com} {path }]" url="https://api.zilliqa.com"
WARN[0189] retry GetBlockByNumber                        attempts=2 number=1002025 sleep=32.644065751s symbol=ZIL
```

Possible solutions:

Set limit for JSON RPC batch request, instead of sending 1K commands, set to 100, but need to test this assumption

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #1406

Changes: 

1. refactor rpc/viewblock sub package
2. break batch json rpc requests into smaller batches

### CODE REVIEW

**Comment by hewigovens** [comment]
will test it locally tmr

File: platform/zilliqa/rpc/client.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+	return block.Header, txs, nil
+}
+
+func (c *Client) GetBlock(number int64) (block Block, err error) {
+	err = c.RpcCall(&block, "GetTxBlock", []string{strconv.FormatInt(number, 10)})
+	return
+}
+
+func makeBatchRequests(hashes []string, batchSize int) (requests []client.RpcRequests) {
```
Comment (vikmeup) [inline platform/zilliqa/rpc/client.go:85]: should we move this into separate package to `client.RpcRequests` ?

File: platform/zilliqa/rpc/client.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		return
+	}
+
+	block, err := c.GetBlock(number)
+	if err != nil {
+		return
+	}
+
+	// Avoid 413 Payload Too Large
+	requests := makeBatchRequests(hashes, 1000)
```
Comment (vikmeup) [inline platform/zilliqa/rpc/client.go:57]: let's use 300 or 500 to be safe

**Review by vikmeup — APPROVED** [review]

File: platform/zilliqa/rpc/client.go (hunk 1 above)
Comment (hewigovens) [reply platform/zilliqa/rpc/client.go:85]: will try to find a generic way for this

**Pull request merged**